### PR TITLE
Dev/ui vuetify

### PR DESCRIPTION
**Pull request by ahnlabb**

### CODE REVIEW

File: src/dashboard/DashboardPage.vue [hunk 1]
```diff
@@ -1,69 +1,80 @@
...
-          | {{ error }}
-    b-row(align-h='end').col-md-6.p-0
-      b-form(inline)
-        b-input-group(append="$").mb-2.mr-sm-2.mb-sm-0
-          b-form-input(v-model="monthlyDonation")
-        b-button(variant="success", @click="$refs.donationSummary.distribute(monthlyDonation)")
-          | Distribute
+  v-layout(row, wrap)
+    v-flex(xs6)
+      v-alert(v-for="error in errors", show, dismissible, variant='warning', @dismissed='dismissedErrors++')
```
Comment (ErikBjare) [inline src/dashboard/DashboardPage.vue]: `variant` remaining

Comment (ErikBjare) [inline src/dashboard/DashboardPage.vue]: `variant` remaining

File: src/dashboard/DonationHistoryComponent.vue [hunk 2]
```diff
@@ -1,45 +1,38 @@
...
+        td
+          a(:href="'https://ropsten.etherscan.io/tx/' + props.item.transaction" target="_blank")
+              | {{props.item.date.toLocaleDateString()}}
+        td
+          a(:href="props.item.url" target="_blank")
+            | {{props.item.creator}}
+        td.text-xs-right
+          | {{props.item.usdAmount}} $
+  div.text-xs-center.pt-2
+    v-btn(v-if="to", variant="outline-secondary", size="sm", :to="to")
```
Comment (ErikBjare) [inline src/dashboard/DonationHistoryComponent.vue]: `variant` remaining

File: src/dashboard/ActivityComponent.vue [hunk 3]
```diff
@@ -1,51 +1,42 @@
...
+  v-card
+    v-data-table(:headers="headers", :items="activities", :pagination.sync='pagination', hide-actions)
+      template(slot='items', slot-scope='props')
+        td
+          a(:href="props.item.url")
+            | {{ props.item.title || props.item.url }}
+        td.text-right
+          | {{ props.item.duration | friendlyDuration }}
+  div.text-xs-center.pt-2
+    v-btn(v-if="to", variant="outline-secondary", size="sm", :to="to")
```
Comment (ErikBjare) [inline src/dashboard/ActivityComponent.vue]: `variant` remaining

File: src/dashboard/App.vue [hunk 4]
```diff
@@ -1,24 +1,29 @@
...
+    v-layout
+      a(href="#/")
+        h1 Thankful Dashboard
+        h3 We're in alpha
         div
-          | Things are not all they could be and will break, but we're working on it.
-  div.row
-    div.col
-      hr
+          | We're still figuring things out. Please report any issues to.
```
Comment (ErikBjare) [inline src/dashboard/App.vue]: Add URL to GitHub issues

File: src/dashboard/DonationHistoryComponent.vue [hunk 5]
```diff
@@ -1,45 +1,38 @@
...
-      td.text-right
-        | {{donation.usdAmount}} $
-  b-button(v-if="to && donationList.length > limit", variant="outline-secondary", size="sm", :to="to")
-    | Show all
+div
+  v-card
+    v-data-table(:headers="headers", :items="donations", :pagination.sync='pagination', hide-actions)
+      template(slot='items', slot-scope='props')
+        td
+          a(:href="'https://ropsten.etherscan.io/tx/' + props.item.transaction" target="_blank")
```
Comment (Powersource) [inline src/dashboard/DonationHistoryComponent.vue]: Maybe for another PR, but this doesn't link correctly for mainnet transactions.

File: src/dashboard/App.vue [hunk 6]
```diff
@@ -2,17 +2,15 @@
 v-app
   v-content
     v-layout
-      a(href="#/")
-        h1 Thankful Dashboard
+      div.pl-3
+        h1 #[a(href="#/") Thankful Dashboard]
```
Comment (ErikBjare) [inline src/dashboard/App.vue]: Use [`router-link`](https://router.vuejs.org/api/#router-link)

File: src/dashboard/DashboardPage.vue [hunk 7]
```diff
@@ -42,7 +42,7 @@ div
         | No creators to show
 
       v-layout(row, wrap)
-        v-flex(v-for="(creator, index) in creators", xs12, sm6, md3)
+        v-flex(v-for="(creator, index) in creators", :key='creator.url', xs13, sm6, md3)
```
Comment (ErikBjare) [inline src/dashboard/DashboardPage.vue]: `xs13`? :open_mouth:

File: src/dashboard/DonationHistoryComponent.vue [hunk 8]
```diff
@@ -12,7 +12,7 @@ div
         td.text-xs-right
           | {{props.item.usdAmount}} $
   div.text-xs-center.pt-2
-    v-btn(v-if="to", variant="outline-secondary", size="sm", :to="to")
+    v-btn(v-if="to", size="sm", :to="to")
```
Comment (ErikBjare) [inline src/dashboard/DonationHistoryComponent.vue]: Why `:to="to"`?

File: src/dashboard/DonationHistoryComponent.vue (hunk 5 above)
Comment (ErikBjare) [reply src/dashboard/DonationHistoryComponent.vue]: Made an issue: https://github.com/SuperuserLabs/thankful/issues/65

**Pull request merged**